src/components/Articles.jsx

```diff
@@ -81,7 +83,13 @@ const Articles = () => {
         {articles.map((article) => {
           return (
             <li key={article.article_id}>
-              <h2 className="articleTitle">{article.title}</h2>
+              <Link
+                to={`/articles/${article.article_id}`}
+                state={{}}
```

Comment:
It looks like you don't need the _state_ attribute in your Link tag so you can get rid of it :)